src/main/java/ch/njol/skript/effects/EffContinue.java

```diff
@@ -25,25 +25,33 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.LoopSection;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.TriggerItem;
-import ch.njol.skript.lang.TriggerSection;
-import ch.njol.skript.lang.parser.ParserInstance;
-import ch.njol.skript.sections.SecLoop;
-import ch.njol.skript.sections.SecWhile;
 import ch.njol.util.Kleenean;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 @Name("Continue")
-@Description("Skips the value currently being looped, moving on to the next value if it exists.")
-@Examples({"loop all players:",
-		"\tif loop-value does not have permission \"moderator\":",
-		"\t\tcontinue # filter out non moderators",
-		"\tbroadcast \"%loop-player% is a moderator!\" # Only moderators get broadcast"})
+@Description("Skips the value currently being looped, moving on to the next value/iteration if it exists/true.")
```

Comment:
```suggestion
@Description("Immediately moves the (while) loop on to the next iteration.")
```
I think this would be better